paymentsheet/src/main/java/com/stripe/android/paymentsheet/address/TransformAddressToElement.kt

```diff
@@ -76,7 +77,7 @@ internal class FieldSchema(
 )
```

Comment:
Since `TransformSpecToElement` is changed to a class, could this be a class too?

Reply:
I am open to more discussion on this.  TransformSpecToElement was changed to a class because it needed a member variable ResourceRepository.   This class doesn't have any member variables, so really all the methods would just be static in it, so I am inclined to just leave it as is.